src/forked/marked/Gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp = require('gulp');
```

Comment:
This Gulpfile will never be ran, as we're referencing the `lib/marked` from everything else, and using our own build process.